clients/cobol-lsp-vscode-extension/src/task/script/compile.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2022 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Broadcom, Inc. - initial API and implementation
+ */
+export const COMPILE_JCL_HEADER: string =
+`//*
+//*  COBOL LIBRARIES
+//*
+//*
+//******************************************************************
+//*  COMP - COMPILE THE COBOL PROGRAM
+//******************************************************************
+//*
+//COMP     EXEC PGM=IGYCRCTL,REGION=0M
+//*  PARM=('LIST,APOST,NOSEQ,LIST,MAP,XREF,SOURCE,LP(64)')
+//*  PARM=('LIST,APOST,NOSEQ,LIST,MAP,XREF,SOURCE,DATA(24)')`;
```

Comment:
These are commented out and hence do not needed. But I believe we wanted some of the options by default. Also we should allow setting them through the task configuration.